### PR TITLE
feat: migrate responsive web design projects

### PR DESCRIPTION
**Pull request by nhcarrigan**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Related to https://github.com/freeCodeCamp/freeCodeCamp/issues/44664

This PR brings over the Responsive Web Design projects from CodePen to .rocks

### CODE REVIEW

**Review by ShaunSHamilton — COMMENTED** [review]
Would you mind adding a `sample.env` to each? Otherwise, it might be better to open a different PR that changes our current config to make use of a standard `sample.env` (if the app does not contain one)

Also, where does this fit in with necessary changes to https://github.com/freeCodeCamp/demo-projects-nginx-config ?

**Comment by nhcarrigan** [comment]
> Also, where does this fit in with necessary changes to https://github.com/freeCodeCamp/demo-projects-nginx-config ?

That's where it is. I was looking for it in the main nginx-config repo. 😅

File: apps/personal-portfolio/index.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        &copy; Created for
+        <a href="https://www.freecodecamp.com/" target="_blank"
+          >freeCodeCamp <i class="fab fa-free-code-camp"></i
+        ></a>
+      </p>
+    </footer>
+
+    <!-- END FOOTER SECTION -->
+  </body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (ShaunSHamilton) [inline apps/personal-portfolio/index.html:217]: Finally, do we want to include this?

Comment (nhcarrigan) [reply apps/personal-portfolio/index.html:217]: We include it on the CodePen projects, I don't see why we wouldn't include it here.

Comment (ShaunSHamilton) [reply apps/personal-portfolio/index.html:217]: Well, we plan on pushing this out when Campers will no longer use that. So, it will be kind of random.

Comment (nhcarrigan) [reply apps/personal-portfolio/index.html:217]: Right. I completely forgot that we're moving the tests to be integrated. 🤦🏻‍♀️ 

Don't mind me, it's been a weird morning. I'll remove these.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

The only thing I noticed, it's not necessarily related - but on the portfolio project, there are codepen links to a lot of the other projects. We could maybe change those to the .rocks versions. Some of them aren’t on .rocks - some are. Not necessary on this PR.

**Pull request merged**